### DISCUSSION TITLE
ERROR 500: Plugin installer failed with 'u' - RC5

### DISCUSSION THREAD

**Discussion by PathTracer75** [calibx/msfsdeck#34] [General]
HI, I have everything updated but the system gives me this error: ERROR 500: Plugin installer failed with 'u'
During the installation process the Loupdeck Live turns off and then I get the error.

Thanks!

**Comment by calibx** [comment]
Hi,
Yes I think there is a problem with the update but I don't know why.
To correct this you need to uninstall the previous installation before upgrading, so delete the folder %userprofile%\AppData\Local\Loupedeck\Plugins\MSFSPlugin
Then you can double click the new .lplug4

**Reply by PathTracer75** [reply]
Hi and thanks for getting back.

I did as you've recommended and I had no error but the cmd window kept itself opened and I had to turn it off manually.
During the installation process I kept also turned off the Loupdeck Live. Once everything done, I started again the Loupdeck app and imported manually the profile in the MSFS plugin. That is correct right?

The only error I saw in the log was : 2022-01-23T15-50-30-9893 |   1 | WARN  | 'PluginConfiguration.xml' file not found for 'Msfs' plugin

By the way, the Cabin Light command seems inverted, when the game starts it turns to Disable Cabin mode when it should be the opposite.

Thanks a lot for your efforts and the great work you're doing!

**Reply by calibx** [reply]
Hi,

I hope the install/upgrade will be fixed for the new release. Still looking what I am doing wrong.
For the cabin light, I've the problem with some planes where they are inverted. What plane did you use ?

**Reply by PathTracer75** [reply]
For now I found this on the new Kodiak 100 and I think also on the default TBM.

Thanks again.

**Reply by calibx** [reply]
Ok I confirm for the TBM, but I don't have the Kodak. Caould you give me the exact name of the aircraft ?
In the debug folder of the plugin there is a "aircraft" input for that.

**Comment by PathTracer75** [comment]
The Kodiak's folder has this name in my community folder:  sws-aircraft-kodiak-wheels
Is one of the best planes for now, just for your info: https://simworksstudios.com/kodiak-100-wheels-for-msfs.html

**Comment by calibx** [comment]
Oh it looks great !
The name I need is the "title" in the manifest.json file in the plane folder or the one displayed on the loupedeck on the debug aircraft input

**Comment by PathTracer75** [comment]
Here you are:

"title": "Kodiak 100",

:)

**Comment by calibx** [comment]
Ok the two are fixed for the next release, next weekend I think

**Reply by PathTracer75** [reply]
Thanks again for your efforts! I've got the Loupdeck Live last week and is changing my life on work and gaming!

**Comment by calibx** [comment]
RC6 is out with the fixes !

**Comment by PathTracer75** [comment]
Thanks for the update, I am looking forward to test it later today.

I'll let you know and thanks so much for your efforts!


-------- Message d'origine --------
De : calibx ***@***.***>
Date : 29/01/2022 11:53 (GMT+01:00)
À : calibx/msfsdeck ***@***.***>
Cc : PathTracer75 ***@***.***>, Author ***@***.***>
Objet : Re: [calibx/msfsdeck] ERROR 500: Plugin installer failed with 'u' - RC5 (Discussion #34)


RC6 is out with the fixes !

—
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcalibx%2Fmsfsdeck%2Fdiscussions%2F34%23discussioncomment-2071458&data=04%7C01%7C%7C3acceada2b8e45cf4b8308d9e3159602%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637790504120682356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=fZnY0iUhKraEDnOCKQ6vlaxK2xs%2F7yPaNclQP9Bmk4M%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAXNQTUL3U3TNTKUKKLQAIVLUYPBKTANCNFSM5MSSMX3Q&data=04%7C01%7C%7C3acceada2b8e45cf4b8308d9e3159602%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637790504120682356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=igm2eGN1SN%2BK1EIBu3xlWQZksl0slDIaeaQ%2FGMyhewM%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C3acceada2b8e45cf4b8308d9e3159602%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637790504120682356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Rm0EYnl%2FETvdU2I%2BSG0uowriKm81ihPqQmL%2BzKOgEIg%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C3acceada2b8e45cf4b8308d9e3159602%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637790504120682356%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qio1V8%2B8yXy9SSyDlCK6A%2FbQ7QuIyTIVR%2BGAzzMUB9E%3D&reserved=0>.

**Comment by PathTracer75** [comment]
Hi!

Here is my feedback:

Installation was ok except of the command window which rest opened and I have to close it manually, I guess is not normal.
On the Kodiak 100 the Cabin Lights button seems still inverted unfortunately.
All the rest works perfectly and the ATC implementation is cool, great idea and great job!

Thanks again for your contribution! :)

**Reply by calibx** [reply]
Hi,

Strange for the light ... can you put in your workspace the input named aircraft in the debug category ? It will display what is the ingame name of the plane to confirm it.

**Reply by PathTracer75** [reply]
![20220130_194719](https://user-images.githubusercontent.com/98240977/151713272-aa8eb1db-92e9-4c22-8802-f123b19a73cc.jpg)
![20220130_194636](https://user-images.githubusercontent.com/98240977/151713274-30ac5310-307d-4847-96fe-9a38b2c905c8.jpg)


Hi!

You were right, the name of the plane is different than the one I gave to you some days ago. Thanks to the debug aircraft tool now we have the complete ones. The Kodiak 100 is installed in the sim in 2 variations, a normal and cargo one.

So as you can see from the photos, the names are:

1. Kodiak 100 Aviation Sans Frontieres
2. Kodiak 100 Cargo Pod, Aviation Sans Frontieres

Hope this helps!

**Reply by PathTracer75** [reply]
I am having a big doubt now....
The data I sent before were relative to the default plane loaded in the sim. But the plane has different liveries and for each livery there is a different name in the Debug Aircraft! I am sorry for this... How did you deal with the other aircrafts and their liveries at this point?

**Comment by calibx** [comment]
No problem, I will handle it with a simple contains instead of equals !
It wiil be fixed for the next release.

**Reply by calibx** [reply]
RC7 is out and it should fix it !
Note : Since RC6 you need to activate WASM in FSUIPC because I add LVAR data :
![image](https://user-images.githubusercontent.com/17730126/151717338-c7cf95a2-01da-469b-9aa1-a03acf1fd4a8.png)

**Comment by PathTracer75** [comment]
Wow, that was fast!

I have an hour of programmed flight now. Once finished I'll install it and try.

I'll leave my feedback again.

Thanks!


-------- Message d'origine --------
De : calibx ***@***.***>
Date : 30/01/2022 21:42 (GMT+01:00)
À : calibx/msfsdeck ***@***.***>
Cc : PathTracer75 ***@***.***>, Author ***@***.***>
Objet : Re: [calibx/msfsdeck] ERROR 500: Plugin installer failed with 'u' - RC5 (Discussion #34)


RC7 is out and it should fix it !

—
Reply to this email directly, view it on GitHub<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcalibx%2Fmsfsdeck%2Fdiscussions%2F34%23discussioncomment-2077719&data=04%7C01%7C%7Cd9dcc7167e6645d523ed08d9e430fd1a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637791721327666096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Hl5gdvW1Q0b1FHxcLUfds5pDL%2FquTQJFO2GdF%2FFLfAk%3D&reserved=0>, or unsubscribe<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAXNQTUJDRNIH35SYYSJVYULUYWPCDANCNFSM5MSSMX3Q&data=04%7C01%7C%7Cd9dcc7167e6645d523ed08d9e430fd1a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637791721327666096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=4UFTxaamIyH5sg9sx2iaQcM2ybQzwz%2B5bq9sl5V4f50%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7Cd9dcc7167e6645d523ed08d9e430fd1a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637791721327666096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=HulOAvHBVZa%2FEeHpMgz6uGR60QNtliJO7uwwX2kluCQ%3D&reserved=0> or Android<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7Cd9dcc7167e6645d523ed08d9e430fd1a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637791721327666096%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=FdwATbTiW0zyuBH6s%2FWF11M06jrvktCtEzZmcetJMqk%3D&reserved=0>.

**Reply by PathTracer75** [reply]
Installed the RC7 and it works perfectly!
Thanks again for your fast support and good work and I'll keep you updated in case I find something wrong.

Take care!
